app/src/main/java/org/simple/clinic/widgets/DividerItemDecorator.kt

```diff
@@ -0,0 +1,60 @@
+package org.simple.clinic.widgets
```

Comment:
Instead of trying to show/hide dividers manually in the view, `RecyclerView` will automatically show dividers between the items. This is to avoid having to test and maintain the show/hide divider logic because this is not what we are trying to solve. So switched to using `DividerItemDecorator`.

Reply:
Cool.